ui/packages/catalog-ui-search/src/main/webapp/js/cql.js

```diff
@@ -23,7 +23,7 @@ const comparisonClass = 'Comparison',
   logicalClass = 'Logical',
   spatialClass = 'Spatial',
   temporalClass = 'Temporal',
-  timePatter = /([0-9]{4})(-([0-9]{2})(-([0-9]{2})(T([0-9]{2}):([0-9]{2})(:([0-9]{2})(\.([0-9]+))?)?(Z|(([-+])([0-9]{2}):([0-9]{2})))?)?)?)?/i,
+  timePatter = /((([0-9]{4})(-([0-9]{2})(-([0-9]{2})(T([0-9]{2}):([0-9]{2})(:([0-9]{2})(\.([0-9]+))?)?(Z|(([-+])([0-9]{2}):([0-9]{2})))?)?)?)?)|^'')/i,
```

Comment:
✏️ I'd like to see this renamed to `timePattern` if it wouldn't involve too many changes

Reply:
Done, good call!